deployment/postgres-init-db/sql/tables/merlin/external_events/plan_derivation_group.sql

```diff
@@ -0,0 +1,25 @@
+create table merlin.plan_derivation_group (
+    plan_id integer not null,
+    derivation_group_name text not null,
+    last_acknowledged_at timestamp with time zone default now() not null,
+
+    constraint plan_derivation_group_pkey
+      primary key (plan_id, derivation_group_name),
+    constraint plan_derivation_group_references_plan_id
+      foreign key (plan_id)
+      references merlin.plan(id),
+    constraint plan_derivation_group_references_derivation_group_name
+      foreign key (derivation_group_name)
+      references merlin.derivation_group(name)
```

Comment:
Critical: `on delete` behavior is missing. This notable as the first constraint should have `on delete cascade` and the second should have `on delete restrict`.

**This is a reoccurring issue throughout the PR**

Reply:
This is now addressed in [add on delete behavior](https://github.com/NASA-AMMOS/aerie/commit/690321653d71eb8cb59db52c37a26b02b3e59cf1).

Reply:
Adjacent matter that I noticed while reviewing that commit: for foreign keys that are linked using a text field, they should include `on update cascade`. This avoids issues with renaming these columns in the (for example, when fixing a typo in an external source type name).

Reply:
Noted. This has now been updated in [add `on update cascade` to foreign keys](https://github.com/NASA-AMMOS/aerie/commit/252f5e950ac2f59d4891a9ab32d82d881ff54d26). I did add an `on update cascade` in `external_event.sql`, in the foreign key that points to the compound, text-based primary key for `external_source` (which is `key, derivation_group_name`); that shouldn't cause issue, right?

Reply:
No that should be fine.